maquetado/scss/application.scss

```diff
@@ -0,0 +1,34 @@
+$blue: #00ADEE;
```

Comment:
It is better to put the colors in a different file so that they can be imported in multiple scss files if needed. Here it doesn't make any difference as you have only one file of scss, but take that into consideration for the future 😄